extensions/sql-migration/src/constants/strings.ts

```diff
@@ -434,7 +434,7 @@ export const SERVER = localize('sql.migration.server', "Server");
 export const USERNAME = localize('sql.migration.username', "Username");
 export const SIZE = localize('sql.migration.size', "Size (MB)");
 export const LAST_BACKUP = localize('sql.migration.last.backup', "Last backup");
-export const DATABASE_FOR_MIGRATION = localize('sql.migration.database.migration', "Database(s) for Migration");
+export const DATABASE_FOR_MIGRATION = localize('sql.migration.database.migration', "Databases for Migration");
 export const DATABASE_MIGRATE_TEXT = localize('sql.migrate.text', "Select database(s) that you want to migrate to Azure SQL");
```

Comment:
for consistency, I think the label "Databases for Migration" should be sentenced cased as 'Databases for migration'

Reply:
Since it is a title, shouldn't the "Migration" part be capitalized?

Reply:
sorry it's ok to leave for now, after looking at other parts of ADS, it seems inconsistent how to apply the capitalization.